Bottom: d31387d87349ffc6945e7fe66a29bad8d0bd1140
Top:    b21572537b1edaa13dee89617b72b9a1a7463a5f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-11-21 21:21:37 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
